custom_components/plugwise/sensor.py

```diff
@@ -2,7 +2,7 @@
 
 import logging
 
-from homeassistant.const import ENERGY_KILO_WATT_HOUR, PERCENTAGE
+from homeassistant.const import ENERGY_KILO_WATT_HOUR, PERCENTAGE, ATTR_UNIT_OF_MEASUREMENT, ATTR_DEVICE_CLASS, ATTR_ICON
```

Comment:
Black formatting

Reply:
Blacked everything in latest commit (tnx!)